resources/js/src/app/components/itemList/filter/ItemFilter.js

```diff
@@ -37,6 +37,18 @@ Vue.component("findologic-item-filter", {
             selectedFacets: state => state.itemList.selectedFacets,
             isLoading: state => state.itemList.isLoading
         }),
+
+        selectedValuesCount() {
+            let count = 0;
+
+            this.facet.values.forEach(function(value) {
```

Comment:
How about this? It should yield the same result (I didn't test it), but is easier to read in my opinion.

```js
const selectedFacets = this.facet.values.filter(function (value) {
    return value.selected;
});

return selectedFacets.length;
```

Reply:
Also don't we already have a helper `getSelectedFilters` for this purpose, why not reuse the existing logic? This way we already could utilize `getSelectedFilters().length` in the template.

Reply:
getSelectedFilters returns the selected filters, not the selected values for each filter. So extra stuff would have to be done which would end up being messier than this, unfortunately.

Reply:
Ah I see.